### ISSUE TITLE
Can't get overviewer-0.16.12 to run

### ISSUE THREAD

**Issue by AMGitsKriss** [overviewer/Minecraft-Overviewer#1894]
I just pulled the compiled 0.16.12 from the website, but when I attempt to run it, I get the following error:

```
  File "overviewer.exe", line 1
SyntaxError: Non-UTF-8 code starting with '\x90' in file overviewer.exe on line 1, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

Defender flagged the app too, but I've seen #1893 and added an exception.

**Comment by CounterPillow** [comment]
Works for me (the 64 bit version at least), try redownloading the zip and extracting to a clean folder that you add to Defender's exceptions beforehand.

**Comment by AMGitsKriss** [comment]
It was the x64 version I was trying. Redownloading made no difference. I got around this by compiling it myself. 

It's not strictly relevant to my original comment, but I did have some odd behaviour while building,
I last compiled this myself back in April with no problems. This time, for some reason, it was trying to use 32-bit msbuild with 64-bit python, but that was easily fixed by forcing them both onto the same architecture.

Feel free to close this if it's no longer relevant. :)
